.github/workflows/install_docs_deps.sh

```diff
@@ -1,3 +1,6 @@
+npm install katex
+PATH="${PATH}:$(pwd)/node_modules/.bin"
```

Comment:
I think env var setup wont work if we call it like `bash install_docs_deps.sh` and maybe it is not necessary if npm is already present in github actions...

Reply:
sudo mode on for `katex` 😁